addons/web/static/src/views/fields/selection/selection_field.scss

```diff
@@ -1,7 +1,7 @@
 body:not(.o_touch_device) .o_field_selection {
     &:not(:hover):not(:focus-within) {
         & select:not(:hover) {
-            background: none !important;
+            background: none;
```

Comment:
DO we need to remove the whole background or just the background-image could be enough ?

Reply:
@morm-odoo kind reminder :slightly_smiling_face: